### PR TITLE
Item image updates

### PR DESCRIPTION
**Pull request by dualcnhq**
## Changes:

- Add etree mediatype checking for waveform thumbnail background
- Change waveform styling and deemphasize background
- Decouple item-image component

TODO:
- implement changes for item-image in tile-list component as well
- add dynamic sizing for the item-image

### CODE REVIEW

File: src/tiles/item-image.ts [hunk 1]
```diff
@@ -36,10 +39,18 @@ export class ItemImage extends LitElement {
...
+      this.isWithWaveformMediatype = withWaveformMediatype.includes(
+        this.model?.mediatype
+      );
+  }
+
   render() {
     return html`
       <div class=${ifDefined(this.imageBoxClass)}>
-        ${this.model?.mediatype === 'audio'
+        ${this.isWithWaveformMediatype
```
Comment (iisa) [inline src/tiles/item-image.ts:53]: you can do a getter here do you don't need to bloat state, like: 
```
get isWithWaveformMediatType() {
return this.model?.mediatype === 'audio'
}
```

File: src/tiles/item-image.ts [hunk 2]
```diff
@@ -38,201 +29,41 @@ export class ItemImage extends LitElement {
...
-  private get backgroundImageTemplate() {
-    return html`
-      <div
-        class=${this.imageClass}
-        style="background-image:url(${this.imageSrc})"
-      ></div>
-    `;
+  private get isWithWaveformMediatype() {
+    return (
+      this.model?.mediatype === 'audio' || this.model?.mediatype === 'etree'
```
Comment (iisa) [inline src/tiles/item-image.ts:44]: wow, TIL `etree` is a mediatype

**Review by iisa — COMMENTED** [review]
thank you for the care in separating image logic. 👍 

will look for unit tests & qa steps

File: src/tiles/item-image.ts (hunk 2 above)
Comment (iisa) [reply src/tiles/item-image.ts:44]: Q: some `etree` / lam items have primary images. what shows up on their tile? waveform bg w/ primary image?

**Comment by dualcnhq** [comment]
closing this branch - moving this to: https://github.com/internetarchive/iaux-collection-browser/pull/57

File: src/tiles/item-image.ts (hunk 2 above)
Comment (dualcnhq) [reply src/tiles/item-image.ts]: were currently just detecting if the loaded image is a waveform based on the aspectRation... so if we detected it's a waveform, then we'll load the waveform with background, but if it has an actual image cover/album cover of some sort then we'll load that image instead

**Pull request closed without merging**